src/lib/renderer-types.ts

```diff
@@ -47,6 +47,15 @@ export interface Renderer {
    */
   canvas: HTMLCanvasElement;
 
+  /**
+   * After the canvas changes shape, this method instructs the Scene to adjust
+   * the offset and canvas properties (width, height) to accommodate.
+   *
+   * @param fixedWorldPoint Optional world point to preserve relative to the
+   * canvas frame. Defaults to the world origin (0,0).
+   */
+  resize: (fixedWorldPoint?: {x: number, y: number}) => void;
```

Comment:
Adding this `resize()` method to allow users to explicitly request an update to the `offset` and the canvas's drawable size (its `width` and `height` properties). Without this, the canvas would simply stretch.